Dockerfile.test

```diff
@@ -12,7 +12,8 @@ RUN apt-get update \
       supervisor \
       curl \
       bridge-utils \
-      net-tools
+      net-tools \
+      procps
```

Comment:
why do we need procps?

Reply:
There was no way to know what processes were running in the container. So I added this to get access to ps command. This only affects the test container. This does not affect the restapi prod container built using buildimage repo

Reply:
Although, this is not related to the PR, this package is required in test docker, as the work-around is to otherwise install using apt-get.